### PR TITLE
Bump 2024.11.2

### PR DESCRIPTION
**Pull request by soapy1**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by soapy1** [comment]
@conda-forge-admin, please rerender

**Comment by trallard** [comment]
Since this is a version bump, I updated the checklist at the top to actually reflect the changes since both of these were checked

> Bumped the build number (if the version is unchanged)
> Reset the build number to 0 (if the version changed)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -81,15 +81,15 @@ outputs:
...
-        - pydantic <2.0a0
+        - pydantic >=2.0
         - pyjwt
         - python >={{ python_min }}
         - python-docker
         - python-multipart
         - pyyaml >=6.0.1
         - redis-py
         - requests
-        - sqlalchemy <2.0a0
```
Comment (trallard) [inline recipe/meta.yaml:92]: Is there a reason why this pin is different from what [we have in our repository `meta.yaml`](https://github.com/conda-incubator/conda-store/blob/f8875ca977fe8309b521e23a5582af7cfd64ec6b/recipe/meta.yaml#L91-L92)?
There we have `- sqlalchemy >=2.0,<2.1` same as in the `pyproject.toml`.

We should make sure that these three files (barring any exceptional circumstances).

Comment (soapy1) [reply recipe/meta.yaml:92]: Good catch, updated!

**Review by trallard — APPROVED** [review]

**Pull request merged**